examples/bench/main.go

```diff
@@ -0,0 +1,1020 @@
+// Copyright 2016 Hajime Hoshi
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"errors"
+	"flag"
+	"fmt"
+	"image"
+	"image/color"
+	"image/draw"
+	_ "image/png"
+	"math"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/hajimehoshi/ebiten"
+	"github.com/hajimehoshi/ebiten/ebitenutil"
+	"github.com/hajimehoshi/ebiten/examples/resources/images"
+	emath "github.com/hajimehoshi/ebiten/internal/math"
+	"github.com/hajimehoshi/ebiten/internal/testflock"
+)
+
+var noErr = errors.New("clean")
+
+type benchmark struct {
+	name   string
+	fn     func(b *testing.B, screen *ebiten.Image)
+	result testing.BenchmarkResult
+}
+
+func main() {
+	flag.Parse() // so -test.benchtime can work
+	testflock.Lock()
+	defer testflock.Unlock()
+
+	// Run an Ebiten process so that (*Image).At is available.
+	f := func(screen *ebiten.Image) error {
+		return benchmarks(screen)
+	}
+	if err := ebiten.Run(f, 320, 240, 1, "Test"); err != nil && err != noErr {
+		panic(err)
+	}
+	for _, b := range benchList {
+		fmt.Printf("%s\n", b.name)
```

Comment:
Why not `fmt.Println(b.name)`?

Reply:
Habit, I'm not used to Println existing yet. :)

Reply:
But you use `fmt.Println` at the next line :-)